GitHub.Collectors/Telemetry/GitHubApplicationInsightsTelemetryClient.cs

```diff
@@ -40,7 +40,9 @@ public override void TrackRequest(string identity, string apiName, string reques
                     { "RateLimitLimit", rateLimitLimit },
                     { "RateLimitRemaining", rateLimitRemaining },
                 };
+
                 this.TrackEvent("RateLimiter", properties);
+                this.logger.LogInformation("Identity {Identity} has {RateLimitRemaining} of {RateLimitLimit} remaining", identity, rateLimitRemaining, rateLimitLimit);
```

Comment:
Something like this? Otherwise, the message will read *"Identity foo has 2 of 4 remaining"*.
```suggestion
                this.logger.LogInformation("Rate limit: Identity {Identity} has {RateLimitRemaining} of {RateLimitLimit} remaining", identity, rateLimitRemaining, rateLimitLimit);
```